/b_scan_publisher:
  ros__parameters:
    azimuth_offset: 0
    end_azimuth: 400
    end_bin: 2856
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    radar_ip: 192.168.0.1
    radar_port: 6317
    start_azimuth: 0
    start_bin: 0
    use_sim_time: false
